sender.go

```diff
@@ -10,7 +10,12 @@ import (
 // When all messages have been successfully produced, Kasper updates the consumer offsets of the input partitions
 // and resumes processing.
 type Sender interface {
+
+	// Send adds a message to an batch that is send when Process returns or when Flush is called
```

Comment:
typos: Send appends a message to a slice held by the sender instance. These messages are sent in bulk when Process() returns or when Flush() is called.